src/Loader.php

```diff
@@ -147,6 +144,19 @@ public function get_services(): array
         return $this->services;
     }
 
+    /**
+     * Load blocks from Theme.
+     */
+    public static function add_blocks(): void
+    {
+        if (!Planet4Blocks::is_active()) {
+            return;
+        }
+
+        new MasterBlocks();
```

Comment:
The sonarcloud is complaining here -

> Either remove this useless object instantiation of class "MasterBlocks" or use it

Is it Ok, if we ignore it by adding a NOSONAR comment?
`new MasterBlocks();//NOSONAR`
cc: @lithrel

Reply:
Yes :) either that or making those class methods static, anything works fine :+1: